src/reanimated2/Hooks.js

```diff
@@ -270,22 +270,25 @@ function styleUpdater(viewDescriptor, updater, state, maybeViewRef) {
   state.last = Object.assign({}, oldValues, newValues);
 
   if (Object.keys(diff).length !== 0) {
-    updateProps(viewDescriptor, diff, maybeViewRef);
+    updateProps(viewDescriptor, diff, maybeViewRef, adapters);
   }
 }
 
-export function useAnimatedStyle(updater, dependencies) {
+export function useAnimatedStyle(updater, dependencies, adapters) {
   const viewDescriptor = useSharedValue({ tag: -1, name: null }, false);
   const initRef = useRef(null);
   const inputs = Object.values(updater._closure);
   const viewRef = useRef(null);
+  adapters = !adapters || Array.isArray(adapters) ? adapters : [adapters];
```

Comment:
```suggestion
  adapters = adapters == null || Array.isArray(adapters) ? adapters : [adapters];
```

Wraps falsy values, leaves `undefined` & `null`.

Reply:
I don't think falsy values should be wrapped. I think it's ok.